### ISSUE TITLE
looks like the preview feature stopped working in the "followed artists" feed

### ISSUE THREAD

**Issue by UmbraVivens** [NightLancer/PixivPreview#19]
that means this page https://www.pixiv.net/bookmark_new_illust.php

**Comment by NightLancer** [comment]
@UmbraVivens I specifically checked the script before the update in my browser(Firefox) and preview was working(all 3 modes). Just now checked on Chrome 95 - works as well. The only case isn't working currently is preview with delay on Daily Rankings, but preview without delay and preview on click works just fine so it's not so critical and I fix it later.

Specify your browser/script manager names and versions, if there are script errors in console(F12) screenshots of them would help as well. Also notice that after one of recent updates minimal versions of browser supported are Firefox 74 and Chrome 80 so check out you have proper ones. I'm using Tampermonkey, but Greasemonkey worked nice previously and hopefully works now as well(but no guarantee). Opera must work too but I'm not testing script there. You can try to temporary disable other scripts(just for test) or change preview setting(either hover/click or set delay), any info would narrow possible cause of problem and quicken answer/solution.

**Comment by UmbraVivens** [comment]
ok i disabled every other script and the preview is still not working in said page, also i just tested and i can't open the settings either.
it still works in other pages
here's a screenshot of the console
![image](https://user-images.githubusercontent.com/54699529/143140021-2a553498-1c29-4205-945d-d0eef57a5650.png)
my browser is Google Chrome 96.0.4664.45 (Build oficial) (64 bits) (cohort: 96_win_45)
i'm using Tampermonkey 4.13

**Comment by NightLancer** [comment]
@UmbraVivens okay, thanks, I already see the problem. There is some element `_ballon-menu-closer` which I don't have on my page(even without any adblock installed) that is mistakenly assumed as arts section. Either you have some extensions that modify pixiv/all sites content as well(aside from userscripts), user css-style(like Stylus), old version of Following page(I had one for like two weeks while some of my script users already had the new one) or you a part of "beta-testing" users which receives updates earlier that others and have even newer one. For crossing-out excessive causes of the problem you can try next:
- ensure you have new page layout like this:
![зображення](https://user-images.githubusercontent.com/19971564/143150536-6f551624-889e-4e61-a8c1-67b8342c0874.png)
- there is no any user-styles at Stylus or similar extension that can modify site layout
- there is no other extension that possibly adds own menu somewhere on the site

Also if you familiar with browser console you can type there next commands(one by one):
`document.querySelectorAll('section ul')`
`document.querySelectorAll('section div ul')`

and make a screen of results such like this:
![зображення](https://user-images.githubusercontent.com/19971564/143151303-b12f43ad-add7-4973-8fb8-aeaa1768f9c2.png)
If you have some extra stuff it's not necessary that you have to disable it if I could find non-conflicting element selector.

UPD: can you also try old [3.10](https://github.com/NightLancer/PixivPreview/raw/9e4052bc8731c8c6f2acd32511407c6afc70d166/PixivPreview.user.js) version of script and tell if it's works? It still had intermediate fix for old/new Following layout and if you have and old one still that may be temporary solution(until you get the new one).

**Comment by UmbraVivens** [comment]
oh yeah i still don't have the new layout, today when trying to remember where was the script's options menu, i noticed i have a "news" thing about the "followed" page being changed soon but i still don't have the redesign.
this is what i get in the console with those commands
![image](https://user-images.githubusercontent.com/54699529/143173466-f30cac44-8206-4d51-8369-c86c71ca727b.png).
also tried logging in in "guest mode" to see what it looks like with absolutely no extensions or scripts and i still had the old layout so i guess i just have to wait or something
![image](https://user-images.githubusercontent.com/54699529/143173802-b30412bb-b259-427a-972f-da0125b9ef9e.png)

**Comment by NightLancer** [comment]
@UmbraVivens hah, so it's turns out to be another pixivs update-based error... As I was not using pixiv so actively I thought I receive updates in most later turn, but seems it's just random. This is very convenient as not having the same page layout(and especially not knowing of this) it is hard(if not impossible) to fix errors. I wish I could have some statistics about old/new layout of my script users, but I haven't... 

Somehow I don't receive announcements notifications and read about changes only now. They started of 5th and now is actually 24th and they not already done... Well, I don't really want to mess up my code with double-cased layout as some parts of it works either on one case or another, and I already implemented new feature like auto-pagination that is incompatible with old layout. I realize that some users may uninstall my script not wanting to wait but well, if they really like it they can reinstall it after receiving new layout. 

As for you I would recommend to stay at the 3.10 version(clickable link in comment above) until you receive new layout as well and disable auto-updates for it if you have ones, it should still work for you if it worked before. 
I'd better leave this issue opened for a while so other can see it and not make the duplicate if they had same problem, and it would be nice if you notify me when you get new layout so I can have reference point when to close it.

Anyway thanks for the feedback, while I was testing code in Chrome after your request I found not critical(therefore not obvious) bug with settings loading and fixed it. _I can't test every possible combination of settings so I rely on reports if something that I missed goes really bad, and when someone already knows how to work in console and make proper screenshots it really helps)_
